PrimeNumbers.py

```diff
@@ -0,0 +1,29 @@
+def is_prime(num):
```

Comment:
The method name `is_prime` suggest that, it'll return a boolean value but this method is returning nothing. It should return `False` when the number is not prime otherwise `True`.

Please remove the `print` statement from your method.

Reply:
I have Updated the issue.